docs/photoniq/fingerprinting/index.md

```diff
@@ -0,0 +1,58 @@
+---
+sidebar_position: 1
+title: Digital Fingerprinting
+---
+
+PhotonIQ Digital Fingerprinting is a managed service that generates persistent digital signatures and visitor IDs for your website users.
+
+## How It Works
+
+Digital Fingerprinting uses advanced fingerprinting techniques to create a unique signature of a user's device. This allows recognizing users across sessions and devices without relying only on cookies.
+
+Some key techniques used:
+
+- Collecting browser signals such as user agent, resolution, fonts, and so on.
+- Leveraging Canvas and AudioContext APIs.
+- Fingerprinting using CPU and Memory.
+- Applying proprietary algorithms and machine learning (ML) models.
+
+When a new user visits your site, Digital Fingerprinting will:
+
+- Generate a **Visitor ID (VID)** to identify that user uniquely.
+- Create a **Signature** to fingerprint the user's device.
+
+The same Visitor ID is recognized using the fingerprint and persists on subsequent visits. The signature is updated to capture any new browser/device attributes. Digital Fingerprinting uses proprietary fuzzy logic and ML models to match fingerprints with high confidence, even if some attributes change. A match score indicates this confidence level.
```

Comment:
Here were talk about using  proprietary fuzzy logic and ML models.

And above we talk about applying proprietary algorithms and ML models.

Probably want both of those to sound the same.

Reply:
Good catch